scaffolder-templates/gitlab-merge-request/template.yaml

```diff
@@ -56,24 +57,23 @@ spec:
         targetPath: '{{ parameters.targetPath }}'
         values:
           name: '{{parameters.name}}'
-
     - id: publish
       name: Publish
       action: publish:gitlab:merge-request
       input:
         repoUrl: '{{ parameters.repoUrl }}'
         projectid: '{{ parameters.gitlab_project_id }}'
```

Comment:
Shouldn't this be updated too?

Reply:
Hi @benjdlambert 

`{{parameters.name}}` is used to replace the name on the skeleton template.

https://github.com/backstage/software-templates/blob/main/scaffolder-templates/gitlab-merge-request/template/catalog-info.yaml

Reply:
So if we don't use projectid in the template we should remove it I think from the parameters block entirely.

Reply:
Otherwise it could cause confusion for people I think :]